styles.css

```diff
@@ -2,325 +2,598 @@ body {
   font-family: 'Inter', Tahoma, sans-serif;
 }
 
-@media all and (max-width: 640px) {
-  .toolbar {
-    display: flex;
-    padding-left: 24px;
-    padding-right: 35px;
-  }
+.navigation {
+  display: flex;
+  padding-left: 24px;
+  padding-right: 35px;
+}
 
-  .toolbar a {
-    flex: 1;
-    font-style: normal;
-    color: #28352f;
-    font-weight: 600;
-    font-size: 20px;
-    line-height: 28px;
-    text-decoration: none;
-  }
+.navigation a {
+  flex: 1;
+  font-style: normal;
+  color: #28352f;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  text-decoration: none;
+}
 
-  .hero {
-    background-image: url(assets/images/background_img.svg);
-    background-size: 100%;
-    height: 90vh;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    margin-bottom: 100px;
-    padding-top: 60px;
-  }
+.hero {
+  background-image: url(assets/images/background_img.svg);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 0;
+  padding-top: 60px;
+}
 
-  .box {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 52px;
-    text-align: center;
-  }
+.hero_content {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+}
 
-  .box_1 {
-    color: #172b4d;
-    margin-bottom: 0;
-  }
+.hero_head {
+  font-style: normal;
+  font-weight: 800;
+  font-size: 48px;
+  line-height: 60px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.hero_head span {
+  color: #36b37e;
+}
+
+.text {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+}
+
+.hero ul {
+  display: flex;
+  width: 30%;
+}
 
-  .box_2 {
-    margin-top: 0;
-    color: #36b37e;
+.hero li {
+  list-style: none;
+  flex: 1;
+}
+
+.projects_header {
+  position: absolute;
+  width: 327px;
+  height: 200px;
+  top: 100px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px;
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.aboutme_head {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  margin: 24px 0;
+}
+
+.works_head {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  margin: 24px 0;
+}
+
+#down_icon {
+  margin-top: 40px;
+}
+
+.grid_container {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto auto auto auto;
+  padding-top: 75px;
+  justify-content: center;
+}
```

Comment:
- [ ] CONGRATULATION :clap: :100: :+1:  your design is perfect you flowed all the requirements of the project as well as the design. However one last touch need to be implemented but your are almost there. The template of this project has 6 cards(projects) but in your design I can just see 3 of them. Kindly add the 3 remaining one, this will helps us to know that you are able to align 6 element using `grid` and by following the template.  
### Your design 
![Screenshot from 2021-12-03 18-47-38](https://user-images.githubusercontent.com/65068771/144642407-7cb76c2d-870a-4a3d-8797-4476beb10dc3.png)
### The template
![Screenshot from 2021-12-03 18-49-12](https://user-images.githubusercontent.com/65068771/144642474-36f0c15d-f483-4839-9814-493e85c48980.png)